libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -124,10 +126,25 @@ singleton AP_Notify alias notify
 singleton AP_Notify method play_tune void string
 singleton AP_Notify method handle_rgb void uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX
 
+include AP_Proximity/AP_Proximity.h
+
+singleton AP_Proximity alias proximity
+singleton AP_Proximity method num_sensors uint8_t
+singleton AP_Proximity method get_object_count uint8_t
+singleton AP_Proximity method get_closest_object boolean float'Null float'Null
+singleton AP_Proximity method get_object_angle_and_distance boolean uint8_t 0 UINT8_MAX float'Null float'Null
+
 include AP_RangeFinder/AP_RangeFinder.h
 
 singleton RangeFinder alias rangefinder
 singleton RangeFinder method num_sensors uint8_t
+singleton RangeFinder method distance_cm_orient uint16_t Rotation'enum Rotation::ROTATION_NONE Rotation::ROTATION_PITCH_7
+singleton RangeFinder method voltage_mv_orient uint16_t Rotation'enum Rotation::ROTATION_NONE Rotation::ROTATION_PITCH_7
+singleton RangeFinder method max_distance_cm_orient uint16_t Rotation'enum Rotation::ROTATION_NONE Rotation::ROTATION_PITCH_7
+singleton RangeFinder method min_distance_cm_orient uint16_t Rotation'enum Rotation::ROTATION_NONE Rotation::ROTATION_PITCH_7
+singleton RangeFinder method ground_clearance_cm_orient uint16_t Rotation'enum Rotation::ROTATION_NONE Rotation::ROTATION_PITCH_7
+singleton RangeFinder method has_data_orient boolean Rotation'enum Rotation::ROTATION_NONE Rotation::ROTATION_PITCH_7
+singleton RangeFinder method get_pos_offset_orient Vector3f Rotation'enum Rotation::ROTATION_NONE Rotation::ROTATION_PITCH_7
 
```

Comment:
@tajisoft san.  Are the following methods likely to be used by you or others in the future?
If so, I think you should add them.

    MAV_DISTANCE_SENSOR get_mav_distance_sensor_type_orient(enum Rotation orientation) const;
    RangeFinder::Status status_orient(enum Rotation orientation) const;
    uint8_t range_valid_count_orient(enum Rotation orientation) const;
    uint32_t last_reading_ms(enum Rotation orientation) const;

Reply:
@muramura I don't use it at this time. I'll make a another PR if I need.